src/firefly/js/ui/TestQueriesPanel.jsx

```diff
@@ -593,6 +640,24 @@ function do2Mass(request) {
     dispatchTableSearch(reqParams);
 }
 
+function doAtlas(request) {
+    console.log('atlas', request);
+    const reqParams = makeTblRequest('ibe_processor', 'ATLAS-' + request[ServerParams.USER_TARGET_WORLD_PT],
+        {
+            [ServerParams.USER_TARGET_WORLD_PT]: request[ServerParams.USER_TARGET_WORLD_PT],
+            mission: 'atlas',
+            //mcenter:true, returned 1 image
+            // TODO Not yet fully working, see AtlasRequestList
+            ds:request.ds1,
+            band: request.band,
+            subsize:0.05,
```

Comment:
do you get more rows with a bigger size?

Reply:
no, subsize is the parameter used for cutout. Number of rows is the resultof the 'band' request, becaue it will end up in the where clause 'band in (...)' and will return in principle max of 5 rows (5 bands) if 'mcen' implementation from Walter is fixed. Currently it isn't fully working as IBE1 ([IRSA-747](https://caltech-ipac.atlassian.net/browse/IRSA-747?focusedCommentId=23640&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23640)).